File: src/main/java/no/nav/fo/veilarbregistrering/registrering/bruker/RegistreringType.java
package no.nav.fo.veilarbregistrering.registrering.bruker;

import no.nav.fo.veilarbregistrering.metrics.Metric;
import no.nav.fo.veilarbregistrering.oppfolging.Oppfolgingsstatus;
import no.nav.fo.veilarbregistrering.sykemelding.SykmeldtInfoData;

public enum RegistreringType implements Metric {
    REAKTIVERING, SPERRET, ALLEREDE_REGISTRERT, SYKMELDT_REGISTRERING, ORDINAER_REGISTRERING;

    @Override
    public String fieldName() {
        return "registreringType";
    }

    @Override
    public String value() {
        return this.toString();
    }
}

File: src/main/kotlin/no/nav/fo/veilarbregistrering/registrering/bruker/RegistreringType.kt
package no.nav.fo.veilarbregistrering.registrering.bruker

import no.nav.fo.veilarbregistrering.metrics.Metric

enum class RegistreringType : Metric {
    REAKTIVERING, ALLEREDE_REGISTRERT, SYKMELDT_REGISTRERING, ORDINAER_REGISTRERING;

    override fun fieldName() = "registreringType"
    override fun value() = this.toString()
}